tests/code/test_spreadsheet.py

```diff
@@ -0,0 +1,194 @@
+                                event = 14,
+                                hoursEarned = 1)
+        assert list(onlyCompletedAllVolunteer("2020-2021").execute()) == []
+        User.create(username = 'solijonovam',
+                    bnumber = 'B00769465',
+                    email = 'solijonovam@berea.edu',
+                    phoneNumber = '732-384-3469',
+                    firstName = 'Madinabonu',
+                    lastName  = 'Solijonova',
+                    isStudent = True,
+                    major = 'Agriculture',
+                    classLevel = 'Sophomore')
+        EventParticipant.create(user = 'solijonovam',
+                                event = 14,
+                                hoursEarned = 1)
+        assert list(onlyCompletedAllVolunteer("2020-2021").execute()) == [('solijonovam', 'Madinabonu Solijonova')]
+        transaction.rollback()
+
+@pytest.mark.integration
+def test_volunteerProgramHours():
+    with mainDB.atomic() as transaction:
+        assert list(volunteerProgramHours().execute()) == ([('Hunger Initiatives', 'neillz', 4.0),  
+                                                            ('Hunger Initiatives', 'khatts', 2.0),
+                                                            ('Berea Buddies', 'bryanta', 0.0),
+                                                            ('Adopt-a-Grandparent', 'khatts', 9.0),
+                                                            ('Hunger Initiatives', 'ayisie', None),
+                                                            ('Hunger Initiatives', 'partont', 5.0)])
+        EventParticipant.create(user = 'qasema',
+                                event = 2,
+                                hoursEarned = 1)
+        assert list(volunteerProgramHours().execute()) == ([('Hunger Initiatives', 'neillz', 4.0), 
+                                                            ('Hunger Initiatives', 'khatts', 2.0), 
+                                                            ('Berea Buddies', 'bryanta', 0.0),
+                                                            ('Adopt-a-Grandparent', 'khatts', 9.0),
+                                                            ('Hunger Initiatives', 'ayisie', None), 
+                                                            ('Hunger Initiatives', 'partont', 5.0),
+                                                            ('Hunger Initiatives', 'qasema', 1.0)])
+        transaction.rollback()
+
+@pytest.mark.integration
+def test_totalVolunteerHours():
+    with mainDB.atomic() as transaction:
+        assert list(totalVolunteerHours().execute()) == [(20.0,)]
+        EventParticipant.create(user = 'qasema',
+                                event = 2,
+                                hoursEarned = 1)
+        assert list(totalVolunteerHours().execute()) == [(21.0,)]
+        transaction.rollback()
+
+@pytest.mark.integration
+def test_getVolunteerProgramEventByTerm():
```

Comment:
Maybe test adding someone to an event here to make sure this is working properly.